src/plugins/vote-out.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable sort-keys */
+import {
+  VoteOut,
+  VoteOutConfig,
+}                 from 'wechaty-voteout'
+
+// import {
+//   DEVELOPERS_ROOM_ID_LIST,
+// }                           from '../rooms-config'
```

Comment:
Shall we remove these comments? Is there any particular reason to keep this?

Reply:
Thanks for the review!

The comments are not removed (yet) is because that this PR is still working on progress...

Reply:
I should create this PR with the `DRAFT` however I forgot.

Will continue to push commits tonight.